### ISSUE TITLE
Playbook failed 

### ISSUE THREAD

**Issue by Davka** [ebbba-org/ansible-role-bigbluebutton#168]
The task includes an option with an undefined variable. The error was: ```bbb_secret``` is undefined

```
ansible-role-bigbluebutton/tasks/check-for-sessions.yml': line 2, column 5
```

**Comment by ChrisKar96** [comment]
I stated the problem in #165 and looking for best way to fix it currently.

For first time running the role (installing bbb) you could run the role with `skip-tags=check-for-sessions` to avoid the check that is failed. The reason is bbb should already be installed for `bbb_secret` to exist, so it is gethered in a latter task, after installation.

**Comment by BrutalBirdie** [comment]
Took me a few solid minutes to understand what's going on and why it's failing.

Why this is failing is this chain of tasks when executing this Role with `bbb_secret` undefined and the default `true` for `bbb_check_for_running_meetings`

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/2a58831486f4889f1cdd382bfa840111a1a2fc7d/defaults/main.yml#L5

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/2a58831486f4889f1cdd382bfa840111a1a2fc7d/tasks/main.yml#L7-L9

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/2a58831486f4889f1cdd382bfa840111a1a2fc7d/tasks/check-for-sessions.yml#L2-L9


I don't even know if this role ever works when `bbb_secret` is undefined at this step.
Because here it gets read from the system.
But this won't prevent this error since it occurs on the first run where nothing is present.

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/83471e86d6ddf982cd7929fe71cdbbac87b8b8ad/tasks/runtimevars.yml#L8-L12

**Comment by BrutalBirdie** [comment]
Created a hotfix PR #169

**Comment by crosscodr** [comment]
Fixed in  8492d45
